### ISSUE TITLE
Inconsistent built-in `<Callout>` layouts between types

### ISSUE THREAD

**Issue by realfish** [shuding/nextra#2352]
The layout metrics, both of vertical and horizontal, of the leading icons in `<Callout>` component may vary per built-in `type` values, due to different emoji fonts.

Would it be a good idea to unify the layout in the built-in theme?

Below is an example in Chrome (Win), where the emoji font is `Segoe UI Emoji` on Windows OS (assigned by Nextra Docs Theme CSS):

![Sample of Windows](https://github.com/shuding/nextra/assets/3247876/fd1e5f4d-6ea0-42a0-be62-122530959f38)

**Comment by B2o5T** [comment]
This happens because only for the info icon we use svg icon instead of emoji 

@realfish should we explicitly setup a width for emoji span element to fix it?

**Comment by realfish** [comment]
@B2o5T Yeah, personally it's a good idea to explicitly set the same width and height for all the Callout icon.

As for 💡 ⚠️ 🚫 used in Callout, the rendered glyph width at a same font size seems to vary among emoji fonts. For 20-px font size, the width is just 20 px in Apple Color Emoji, while ~27.5 px in Segoe UI Emoji and ~24.9 px in Noto Color Emoji.

The rendered height seems to be determined by the line height.
